frontend/src/app/shared/services/snack-bar-manager.ts

```diff
@@ -32,7 +32,7 @@ export class SnackBarManager {
   successFailurePipe(config: SnackBarPipeConfig) {
     const { successMsg, errorMsg, buttonText, ...snackBarConfig } = config;
     return pipe(
-      tap(() => this.openSnackBar(successMsg, buttonText, snackBarConfig)),
+      tap(() => this.snackBar.open(successMsg, buttonText, snackBarConfig)),
```

Comment:
Could you please provide some reasoning for the changes in this file? As far as I remember, `openSnackBar` method was used on purpose, so that the default config with button and auto-closing of message after some time would be applied throughout the app.

Reply:
I missed your comment from above, let me check again first :)

Reply:
It seems that the previous version with `openSnackBar` works just fine on my machine. Could you try once again or try to provide a minimal example on how to replicate the issue?
As mentioned before, I'd rather keep the `openSnackBar` method, as it applies the default config to the base `snackBar.open` that makes it more usable in my opinion.

Reply:
Hmm.. you're right, it was broken at that time for some other reason probably. I've retired these changes.